src/samples/EducationStart/reuseables/WarningText.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
non need to have this component here.
Please import from '\src\components\BaseComponents\WarningText\WarningText.tsx'